srcpkgs/clevis/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'clevis'
+pkgname=clevis
+version=20
+revision=1
+build_style=meson
+hostmakedepends="asciidoc cryptsetup curl jq keyutils pkg-config tpm2-tools"
```

Comment:
why does it need curl during build?

Reply:
It doesn't actually use curl. So no network access or anything, but for one optional feature (tang server support), meson checks if curl is available and only then does it enable that feature. It was only able to find it, when curl was available on the build host. I guess it is not really designed for cross-compilation, but having it available at build time appears to do the trick.

Reply:
ugh trying to be too automagic... might also want to include bash-completion so it does the completions

Reply:
Yes, I've now also included bash-completion.